public/uploads/fake.py

```diff
@@ -1,26 +0,0 @@
-import csv 
```

Comment:
Please ignore public/uploads dir in .gitignore. Otherwise, it consumes unnecessary storage on Github.